torchmetrics/collections.py

```diff
@@ -185,6 +147,51 @@ def persistent(self, mode: bool = True) -> None:
         for _, m in self.items():
             m.persistent(mode)
 
+    def add_metrics(self, metrics: Union[Metric, Sequence[Metric], Dict[str, Metric]],
```

Comment:
```suggestion
    def append(self, metrics: Union[Metric, Sequence[Metric], Dict[str, Metric]],
```
would it be a better name?

Reply:
I think it's fine to keep the name different. If we say `append` it might imply that it won't work as `extend` while here it does.

Reply:
`add_metrics` seems more appropriate for a Collection.